### PR TITLE
Fix checksum error for ROOT5

### PR DESCRIPTION
**Pull request by wmtan**
Fix checksum error in the build of CMSSW_7_5_ROOT5_X.
Also make new ROOT6 checksum known to ROOT5.
Technical and trivial.
Please expedite.
The new ROOT5 checksum will be made known to CMSSW_7_5_X in a separate PR.

### CODE REVIEW

File: DataFormats/PatCandidates/src/classes_def_objects.xml [hunk 1]
```diff
@@ -271,7 +271,8 @@
    <version ClassVersion="11" checksum="1239840459"/>
   </class>
 
-  <class name="pat::PackedCandidate" ClassVersion="19">
+  <class name="pat::PackedCandidate" ClassVersion="21">
+    <version ClassVersion="21" checksum="663492232"/>
```
Comment (slava77) [inline DataFormats/PatCandidates/src/classes_def_objects.xml]: Hi Bill,

there is ``` <version ClassVersion="21" checksum="1075665714"/> ``` in 75X already
and we have queued a ``` <version ClassVersion="22" checksum="691064527"/> ``` into 74X (to be forward ported to 75X/76X).

So, I think this one will need to be ClassVersion="23"

Comment (wmtan) [inline DataFormats/PatCandidates/src/classes_def_objects.xml]: Yes, I saw the version 21 in 75X, so I closed this one temporarily so I could fix it.
I did not know about the version 22.  Thanks for letting me know.

Comment (slava77) [inline DataFormats/PatCandidates/src/classes_def_objects.xml]: sure, the backports to 74X are going through a holding area. So, it's easy to not notice them.

**Comment by davidlange6** [comment]
This should into the mainstream, no?

**Comment by wmtan** [comment]
@davidlange6 #10931 is for the mainstream. Identical to this PR, except using different version number.

**Comment by davidlange6** [comment]
@wmtan -

**Pull request closed without merging**